user-service/src/main/java/rs/edu/raf/banka1/controller/OrderController.java

```diff
@@ -68,7 +89,7 @@ public ResponseEntity<Void> changeRequestStatus(
         @ApiResponse(responseCode = "404", description = "Bad Request"),
         @ApiResponse(responseCode = "500", description = "Internal server error")
     })
-    public ResponseEntity<Void> createLoanRequest(
+    public ResponseEntity<Void> createOrderRequest(
```

Comment:
možda da bude boolean, čisto da znamo da li je uspešno kreiran ili ne.

Reply:
To je Andrejina metoda, pa nisam hteo da menjam. Evo promenio sam sada ali vrv treba dopuniti jos za false.

Reply:
Oke.